website/content/release-notes/_index.md

```diff
@@ -13,7 +13,10 @@ Bug Fixes:
 the service ([Issue 1511](https://github.com/metallb/metallb/issues/1511) [PR 1516](https://github.com/metallb/metallb/pull/1516))
 - Add the AvoidBuggyIPs flag to the IPAddressPool CRD. Converting a CIDR to a range comes with limitation related
 to setting the aggregation length and validating it. ([Issue 1495](https://github.com/metallb/metallb/issues/1495),
-[PR 1515](https://github.com/metallb/metallb/pull/1515)
+[PR 1515](https://github.com/metallb/metallb/pull/1515))
+- Add a valid pem format to the CRDs webhooks instead of the empty placeholder. ([Issue 1501](https://github.com/metallb/metallb/issues/1501),
```

Comment:
Can you add this change in a separate commit?

Reply:
Do you mean to make this change in another PR?

Reply:
nope, I mean a different commit in the same pr (you can have multiple commits in your https://github.com/cyclinder/metallb/tree/fix_invalid_caBundle branch, those will be reflected in separate commits in the pr)